app/src/main/java/de/westnordost/streetcomplete/tangram/MapFragment.java

```diff
@@ -654,6 +654,16 @@ public void setMapOrientation(float rotation, float tilt)
 		onMapOrientation(rotation, tilt);
 	}
 
+	public void setPosition(LngLat position)
```

Comment:
`getPosition` returns a `LatLon` (from osmapi) while this method accepts a `LngLat` (from tangram-es).

The `MapFragment` should use only one position data type on its interface. I tend to say that `LngLat` is a slightly better choice because it has more to do with tangram-stuff than OSM Api data classes.

Reply:
Also, the setter and the getter should be in the same class. Probably in `MapFragment` because this is nothing specific to quests.

Reply:
Are you sure? I checked yesterday and I think I remember it only being used in two instances. A LngLat can easily be converted to a LatLon, there should be an utility class for that.

Reply:
Oh... true. It is only used by the `MainActivity` and the `QuestController`. I thought it would be more 🤷‍♂